src/Symfony/Component/DependencyInjection/Loader/Configurator/ParametersConfigurator.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class ParametersConfigurator extends AbstractConfigurator
+{
+    const FACTORY = 'parameters';
+
+    private $container;
+
+    public function __construct(ContainerBuilder $container)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * Creates a parameter.
+     *
+     * @param string $name
+     * @param string $value
+     *
+     * @return $this
+     */
+    final public function set($name, $value)
+    {
+        $this->container->setParameter($name, static::processValue($value, true));
+
+        return $this;
+    }
+
+    /**
+     * Creates a parameter.
+     *
+     * @param string $name
+     * @param string $value
+     *
+     * @return $this
+     */
+    final public function __invoke($name, $value)
```

Comment:
how is the invoke used? I can't find tests any use of this.

Reply:
for services its tested in php7 fixture but parameters does not seem to have tests for this

Reply:
The feature seems kinda handy. But for IDEs this will make finding service and paramter definitions even harder I assume.

Reply:
Now tested in the php7 fixture. About IDEs, do you have an example? As far as I tested correctly, phpstorm handles them perfectly.